Utils/trigger_test_upload_flow.sh

```diff
@@ -156,6 +156,7 @@ if [ -n "$_gitlab" ]; then
     --form "variables[STORAGE_BASE_PATH]=${_storage_base_path}" \
     --form "variables[OVERRIDE_ALL_PACKS]=${_override_all_packs}" \
     --form "variables[CREATE_DEPENDENCIES_ZIP]=${_create_dependencies_zip}" \
+    --form "variables[TEST_UPLOAD]=true" \
```

Comment:
you can remove it if are making it default (but you should add it to the regular trigger script).